File: stu_spring/webflux/test/auth/src/main/java/com/webflux/auth/domain/auth/exception/InvalidTokenException.java
package com.webflux.auth.domain.auth.exception;

import com.webflux.auth.global.error.exception.ErrorCode;
import com.webflux.auth.global.error.exception.GlobalException;

public class InvalidTokenException extends GlobalException {
    public InvalidTokenException() {
        super(ErrorCode.INVALID_TOKEN);
    }
}

File: stu_spring/kotlin_basic/src/main/kotlin/com/kotlin/test/kotlintest/global/security/exceptions/InvalidTokenException.kt
package com.kotlin.test.kotlintest.global.security.exceptions

import com.kotlin.test.kotlintest.global.exception.GlobalException
import com.kotlin.test.kotlintest.global.exception.enums.ErrorCode

class InvalidTokenException : GlobalException(ErrorCode.INVALID_TOKEN) {
    companion object {
        @JvmField
        val EXCEPTION = InvalidTokenException()
    }
}